samples/grids/data-grid/cell-editing/Services/EmployeeData.cs

```diff
@@ -24,7 +25,7 @@ public class Employee : INotifyPropertyChanged
         public int Index { get; set; }
 
         public DateTime Birthday { get; set; }
-        public List<Productivity> Productivity { get; set; }
+        public ObservableCollection<Productivity> Productivity { get; set; }
```

Comment:
you need to replace "List" with "ObservableCollection" in EmployeeData.cs files that are in all other samples and template/browser (if exits)

Reply:
see commit 2